HidPkg/Include/Protocol/HidIo.h

```diff
@@ -0,0 +1,160 @@
+/** @file
+  HID I/O
+
+  This protocol is used by code, typically drivers, running in the EFI
+  boot services environment to access HID devices.
+
+  Copyright (c) Microsoft Corporation. All rights reserved.
+
+**/
+
+#ifndef __HID_IO_H__
+#define __HID_IO_H__

```

Comment:
To align with the latest edk2 convention, we've move to the following format:

```suggestion
#ifndef HID_IO_H_
#define HID_IO_H_

```

Reply:
Fixed